tools/cldr-apps/js/src/esm/cldrInfo.js

```diff
@@ -12,50 +12,196 @@ import * as cldrTable from "./cldrTable.js";
 import * as cldrText from "./cldrText.js";
 import * as cldrVote from "./cldrVote.js";
 
+import InfoPanel from "../views/InfoPanel.vue";
+
+import { createCldrApp } from "../cldrVueRouter";
+
+let containerId = null;
+let neighborId = null;
+let buttonClass = null;
+
+// Start with panel closed; it will get opened when a Page is chosen
+let panelVisible = false;
+let panelVisibleForPageView = true;
+
 let unShow = null;
 let lastShown = null;
 
+/**
+ * Initialize the Info Panel
+ *
+ * @param {String} cid the id of the container element for the panel
+ * @param {String} nid the id of the neighboring element to the left of the panel
+ * @param {String} bclass the class for "Open Info Panel" buttons
+ */
+function initialize(cid, nid, bclass) {
+  containerId = cid;
+  neighborId = nid;
+  buttonClass = bclass;
+  insertWidget();
+  setPanelAndNeighborStyles();
+  updateOpenPanelButtons();
+}
+
+function insertWidget() {
+  try {
+    const fragment = document.createDocumentFragment();
+    createCldrApp(InfoPanel).mount(fragment); // returns App object "wrapper", currently not saved
```

Comment:
it would be saved if we want to replace it, or if it could load twice in one browser session